.pre-commit-config.yaml

```diff
@@ -25,4 +25,8 @@ repos:
       args: [--markdown-linebreak-ext=md]
     - id: end-of-file-fixer
     - id: mixed-line-ending
-exclude: 'tests/unit/lang/translator/(keys|cs|es|fr|de|pl|it)\.txt$'
+exclude: |
+  (?x)(
+    ^tests/unit/lang/translator/(keys|cs|es|fr|de|pl|it)\.txt$|
+    ^lib/
```

Comment:
Proč je excluded celý lib, ikdyž se styl dříve vynucoval na tom, co nebylo submodul a bylo v této složce?

Reply:
Vyjmenovávat zde subrepa (a všem řikat, že když nějaký přidají, že ho sem musejí přidat) mi nepřišlo jako příliš robustní řešení. Navíc to beru, že všechno v `lib` je third-party code a nemám problém s tím, pokud nebude automaticky formátovaný.

Reply:
Bohužel není vše v lib third-party.

Reply:
Dal bych exclude pouze na to, co bylo dříve submodul a zároveň to není naše a zároveň to nebylo historicky přeformátováno.

Reply:
Souhlas 👍

Reply:
Commit 478008e